### PR TITLE
Additional field tables for easier API integration

### PR DESCRIPTION
**Pull request by shawnawsu**

### CODE REVIEW

File: zendesk.install [hunk 1]
```diff
@@ -636,6 +666,53 @@ function zendesk_update_7106() {
...
+    ],
+    'assignee_uid' => [
+      'type' => 'int',
+      'size' => 'big',
+      'unsigned' => TRUE,
+      'not null' => 0,
+      'default' => 0,
+      'description' => 'The drupal uid of the assignee.',
+    ],
+    'subscription_id' => [
```
Comment (bbujisic) [inline zendesk.install:701]: `license_id`

File: zendesk.install [hunk 2]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+      ),
+      'assignee_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
+        'unsigned' => TRUE,
+        'not null' => 0,
+        'default' => 0,
+        'description' => 'The drupal uid of the assignee.',
+      ),
+      'subscription_id' => array(
```
Comment (bbujisic) [inline zendesk.install:217]: `license_id`

File: zendesk.install [hunk 3]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
         'serialize' => TRUE,
         'binary' => FALSE,
       ),
+      'requester_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
```
Comment (bbujisic) [inline zendesk.install:195]: no size needed, see `user_schema()`

File: zendesk.install [hunk 4]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+        'unsigned' => TRUE,
+        'not null' => 0,
+        'default' => 0,
+        'description' => 'The drupal uid of the requester.',
```
Comment (bbujisic) [inline zendesk.install:199]: "Requester's {users}.uid."

File: zendesk.install [hunk 5]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+      ),
+      'submitter_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
+        'unsigned' => TRUE,
+        'not null' => 0,
+        'default' => 0,
+        'description' => 'The drupal uid of the submitter',
```
Comment (bbujisic) [inline zendesk.install:207]: "Submitter's {users}.uid."

File: zendesk.install [hunk 6]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+      ),
+      'assignee_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
+        'unsigned' => TRUE,
+        'not null' => 0,
+        'default' => 0,
+        'description' => 'The drupal uid of the assignee.',
```
Comment (bbujisic) [inline zendesk.install:215]: "Assignee's {users}.uid."

File: zendesk.install [hunk 7]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+        'type' => 'int',
+        'size' => 'big',
+        'unsigned' => TRUE,
+        'not null' => 0,
+        'default' => 0,
+        'description' => 'The drupal uid of the submitter',
+      ),
+      'assignee_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
```
Comment (bbujisic) [inline zendesk.install:211]: no size

File: zendesk.install [hunk 8]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+        'unsigned' => TRUE,
+        'not null' => 0,
+        'default' => 0,
+        'description' => 'The drupal uid of the requester.',
+      ),
+      'submitter_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
```
Comment (bbujisic) [inline zendesk.install:203]: no size

File: zendesk.install [hunk 9]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+        'unsigned' => TRUE,
+        'not null' => 0,
```
Comment (bbujisic) [inline zendesk.install:205]: 'not null' => FALSE,

File: zendesk.install [hunk 10]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
...
+        'default' => 0,
+        'description' => 'The drupal uid of the submitter',
+      ),
+      'assignee_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
+        'unsigned' => TRUE,
+        'not null' => 0,
```
Comment (bbujisic) [inline zendesk.install:213]: 'not null' => FALSE,

File: zendesk.install [hunk 11]
```diff
@@ -190,6 +190,36 @@ function zendesk_schema() {
         'serialize' => TRUE,
         'binary' => FALSE,
       ),
+      'requester_uid' => array(
+        'type' => 'int',
+        'size' => 'big',
+        'unsigned' => TRUE,
+        'not null' => 0,
```
Comment (bbujisic) [inline zendesk.install:197]: 'not null' => FALSE,

File: zendesk.install [hunk 12]
```diff
@@ -636,6 +666,53 @@ function zendesk_update_7106() {
...
 }
 
+
+/**
+ * Add user uid and subscription id fields to the zendesk_tickets table.
+ */
+function zendesk_update_7107() {
+  $fields = [];
+
+  $fields = [
```
Comment (bbujisic) [inline zendesk.install:676]: copy-pasta

**Review by bbujisic — CHANGES_REQUESTED** [review]

**Comment by shawnawsu** [comment]
I am so sorry, we are reverting way way back to a totally different approach. Just gotta get this delorean up to 88 mph...

**Pull request closed without merging**